app/templates/components/event-table.hbs

```diff
@@ -0,0 +1,58 @@
+<div class = "ui grid container">
+  <div class = "row">
+    <div class = "sixteen wide column">
+      <table class="ui compact  celled table">
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Date</th>
+            <th>Roles</th>
+            <th>Sessions</th>
+            <th>Speakers</th>
+            <th>Tickets</th>
+            <th>Public Url</th>
+            <th>Actions</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>
+              <div><i class="large camera icon"></i></div>
+              <div>Abhinav Khare</div>
+            </td>
+            <td>
+              27-06-2017
+            </td>
+            <td>
+              <ol>
+                <li>abhinav.khare31@gmail.com (Organizer)</li>
+              </ol>
+            </td>
+            <td>
+              <ol>
```

Comment:
There seems to be too much left padding/margin here. Can you make use of https://semantic-ui.com/elements/list.html instead of a normal `ol` to make this look better ? See what all semantic-ui has to offer.